src/components/AdminPanel/MedCentersForm/FormPopup/FormPopup.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { NavLink, useHistory } from 'react-router-dom';
+import { ROUTS } from '@constants/routs';
+import { EditPopupWrapper, PopupContent } from './FormPopupStyle';
+
+interface IFormPopupProps {
+  message?: string;
+  error?: string;
+}
+
+const FormPopup: React.FC<IFormPopupProps> = ({ message }) => {
```

Comment:
do we still need this component?

Reply:
https://github.com/KLIM-nk01/virtual-ambulance/pull/27/commits/35d1051dd92f9591600c90270a824d4eec0360e7#diff-39c650f4e4bb8a9b6263afde19559c2d20b61f9c3ddf989fb32115e2f0bdc392